src/fsharp/FSharp.Compiler.Service/FSharp.Compiler.Service.fsproj

```diff
@@ -1,21 +1,46 @@
-﻿<!-- Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information. -->
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information. -->
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
-    <NoWarn>$(NoWarn);44;62;69;65;54;61;75;62;9;2003;NU5125</NoWarn>
+    <TargetFrameworks>net472;netstandard2.0</TargetFrameworks>
+    <OutputType>Library</OutputType>
+    <NoWarn>$(NoWarn);44;45;54;55;61;62;69;65;75;1204;2003;NU5125</NoWarn>
+    <AssemblyName>FSharp.Compiler.Service</AssemblyName>
     <AllowCrossTargeting>true</AllowCrossTargeting>
-    <DefineConstants>$(DefineConstants);COMPILER_SERVICE_AS_DLL</DefineConstants>
     <DefineConstants>$(DefineConstants);COMPILER</DefineConstants>
     <DefineConstants>$(DefineConstants);ENABLE_MONO_SUPPORT</DefineConstants>
-    <OtherFlags>$(OtherFlags) /warnon:3218 /warnon:1182 /warnon:3390 --times</OtherFlags>
-    <Tailcalls>true</Tailcalls>
-    <!-- .tail annotations always emitted for this binary, even in debug mode -->
-    <NGenBinary>true</NGenBinary>
+    <OtherFlags>$(OtherFlags) /warnon:3218 /warnon:1182 /warnon:3390 --maxerrors:20 --extraoptimizationloops:1 --times</OtherFlags>
```

Comment:
I know this is the same as it's been using before, but wouldn't it be possible to use `WarnOn` and stuff as properties? Feels a bit more normal is all

Reply:
I've looked at this before and I don't believe there's mapping through of `WarnOn` in the msbuild tasks for F#. I'd love to have such a thing, personally.

Reply:
damn, we should really do that

Reply:
@cartermp we don't have a property for it.  If we did, we would have to wait until it had shipped and made it to our lkg build.  I will add an issue, and add the property in a seperate PR  ... but we won't be able to use it here for quite a while ...

Reply:
I made an issue just now, Kevin: #10882